---
layout: archive
title: "Projects"
permalink: /projects/
author_profile: true
redirect_from:
  - /projects
---

<hr>

# Research Projects - Columbia University (Graduate Studies)
## Enhancing [TIGER]([url](https://huggingface.co/spaces/Knowles-Lab/tiger)) Model for Cas13 Off-Target Prediction for Indel gRNA Data
_Sep '23 — Dec '23_
<br>
**Columbia University, New York, NY**
I'm currently improving the TIGER Model for predicting Cas13 off-targets at Columbia University. I'm working on making the model better at handling indel gRNA data by adjusting its setup. My goal is to predict off-target activity more accurately in different genomic situations. I'm using a simple algorithm, the Levenshtein distance, to encode DNA sequences with indels. This work is ongoing, and I'm excited about making Cas13 off-target predictions more reliable and adaptable.
## Predictive Model for Alzheimer’s Disease Prediction using Machine Learning
_Jan '23 — May '23_
<br>
**Columbia University, New York, NY**

In this project at Columbia University, I led an initiative to develop a predictive model for Alzheimer's disease using machine learning. I curated RNA-seq data from Alzheimer patients obtained from public databases and conducted a detailed comparison of gene expression between normal and diseased states using the ANOVA test. Employing advanced techniques, I extracted key Principal Components and strategically stratified the dataset to create robust training and test datasets. I explored various predictive models, prioritizing those with minimal false negatives. This project showcased my expertise in data curation, statistical analysis, and machine learning, emphasizing my ability to extract meaningful insights from complex biological data and contribute to the field of predictive modeling for neurodegenerative diseases.
<br>

## [BRCA-SAE]([url](https://github.com/PankajPradeep/BRCA-SAE)) Multi-Omics Data Integration using Stacked Autoencoders
_Jan '23 — May '23_
<br>
**Columbia University, New York, NY**

At Columbia University, I also took up a very interesting project where I focused on integrated of BRCA-SAE multi-omics data using Stacked Autoencoders. I extracted and processed RNAseq, methylation, and proteomics data for breast invasive carcinoma, from public databases like TCGA and GEO to create the foundation for a comprehensive analysis. Central to this endeavor was the construction of a Stacked Autoencoder model designed to encapsulate multi-omic integrated data within a latent space, allowing for a nuanced understanding of the complex relationships between diverse molecular data types. Leveraging this model, I conducted Feature Attribution analyses to identify biomarkers that serve as pivotal drivers of cancer progression across all datasets.
The subsequent phase involved a meticulous evaluation of the top 20 identified biomarkers. To validate their significance and functionality, I delved into an extensive literature survey, ensuring confirmation of our results. This project not only showcased my proficiency in data integration and machine learning but also underscored my dedication to elucidating meaningful insights in the realm of cancer biology. The combination of technical skills, from data extraction to model construction and literature analysis, positions this project as a testament to my capabilities in unraveling the complexities of multi-omics data for advancing our understanding of breast invasive carcinoma.

# Research Projects -  VIT University (Undergraduate Studies)
## Insights into Phenylketonuria and Suggested Solution to Disorder

_Jan '21 — Jun '21_
<br>
**Vellore Institute of Technology, Vellore, India**
<br>
During this project, I performed a comprehensive literature review to gain a profound understanding of Phenylketonuria (PKU) and the severity of this genetic disorder. I meticulously analyzed the mutations at the genetic level to identify which mutation(s) had the most disruptive or damaging impact on PKU patients. In addition to this, I proposed a novel solution to address PKU and conducted feasibility studies to assess the practicality and effectiveness of this solution. This project not only deepened my knowledge of genetic disorders but also allowed me to explore potential solutions to alleviate the challenges faced by PKU patients.

## In-Silico Analysis of PTEN Variants causing Breast Cancer (Aug 2020 – Feb 2021)

_Aug '20 — Feb '21_
<br>
**Vellore Institute of Technology, Vellore, India**
<br>
This project involved a meticulous analysis of PTEN variants associated with breast cancer. I identified the amino acid changes that were most harmful to the PTEN protein structure and subsequently evaluated the stability of each amino acid change using specialized software tools. Additionally, using PyMol I compared the structural variations between mutated protein structures and the native protein structure when these mutations were induced. To gather relevant genetic information, I utilized databases such as the dbSNP database, Polyphen-2 database server, and SIFT for SNP collection and analysis. This project significantly enhanced my understanding of the genetic factors contributing to breast cancer.

## Vitamin B12: The Energy Vitamin - A Review Paper

_Aug '20 — Jan '21_
<br>
**Vellore Institute of Technology, Vellore, India**
<br>
In this project, I documented a comparative study after conducting an in-depth review of approximately 70 research papers. My focus was on optimizing strategies to enhance the yield of vitamin B12 during industrial production. This project not only contributed to the body of knowledge surrounding vitamin B12 but also explored practical applications for industrial optimization. We drafted an extensive review paper that was submitted to multiple journals for review with a tabular column comparing all the information ever known for producing Vitamin B12 industrially.

## Checking the Effectiveness of Commercially Available Sanitizers on Microbes

_Jan '19 — May '19_
<br>
**Vellore Institute of Technology, Vellore, India**
<br>
As part of my Microbiology course, I investigated the effectiveness of commercially available sanitizers on microbes by collecting swab samples from various sources. I then executed microbiological laboratory techniques, including Spread Plating and Agar-Well Diffusion Methods, to assess the efficiency of these sanitizers. After comprehensive analysis and experimentation, I compiled a final report presenting the findings, which indicated that the sanitizers achieved an efficacy rate of approximately 96-97%. This project provided insights into the real-world application of sanitizers and their impact on microbial communities.
<hr>
